tests/test_target.py

```diff
@@ -146,5 +148,256 @@ def test_image_cache(self):
         third_image = Image(self.file_all_shapes)
         self.assertIsNot(image.pil_image, third_image.pil_image)
 
+
+class ChainTest(unittest.TestCase):
+    """
+    Tests for the chain target (series of steps).
+    """
```

Comment:
We use one-line format for one-line docstrings, i.e. `"""Test for ..."""`. I know this is unit tests but it should be the same everywhere.